### PR TITLE
Add links

### PR DESCRIPTION
**Pull request by alt219**
Add Links, a text mode web browser.

### CODE REVIEW

File: spk/links/Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+SPK_NAME = links
+SPK_VERS = 2.12
+SPK_REV = 1
+SPK_ICON = src/links.png
+
+DEPENDS = cross/links
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Links is a web browser running in text mode.
+RELOAD_UI = no
```
Comment (Dr-Bean) [inline spk/links/Makefile]: Please remove, `no` is the default value.

File: spk/links/Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+DISPLAY_NAME = Links
+STARTABLE = no
+BETA = 1
+
+HOMEPAGE   = http://links.twibright.com/
+LICENSE    = GPLv2
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
+ADDITIONAL_SCRIPTS =
```
Comment (Dr-Bean) [inline spk/links/Makefile]: Please remove if unused

File: spk/links/src/dsm-control.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# Package
+PACKAGE="links"
+DNAME="Links"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:/usr/local/bin:/bin:/usr/bin:/usr/syno/bin"
```
Comment (Dr-Bean) [inline spk/links/src/dsm-control.sh]: Use ${PATH} here

File: spk/links/src/installer.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# Package
+PACKAGE="links"
+DNAME="Links"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:/usr/local/bin:/bin:/usr/bin:/usr/syno/bin"
```
Comment (Dr-Bean) [inline spk/links/src/installer.sh]: Use ${PATH} here

**Comment by Dr-Bean** [comment]
Easy reviewing ;) Can you add a bigger icon? 256*256 is what we're looking for.

**Comment by alt219** [comment]
Thank you, and thanks for your work putting all this together.  Great stuff!

File: spk/links/Makefile [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+SPK_ICON = src/links.png
+
+DEPENDS = cross/links
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Links is a web browser running in text mode.
+DISPLAY_NAME = Links
+BETA = 1
+
+HOMEPAGE   = http://links.twibright.com/
```
Comment (Dr-Bean) [inline spk/links/Makefile]: A suggestion for your next PR: I don't mind the aligning, but let's use the minimum number of spaces to achieve that ;)

**Comment by Dr-Bean** [comment]
Very nice :) Haven't pulled/built yet, but the code looks good enough ;)

File: spk/links/Makefile (hunk 5 above)
Comment (alt219) [inline spk/links/Makefile]: Gotcha, based these files off of a different package, I should have fixed the alignment but it slipped by my eyes.

**Comment by Dr-Bean** [comment]
Slight adjustment to correct status shown in Package Center with df7009def2689b144b7d48d2d3ad29182e1f1279, but otherwise, seems to work fine. Thanks!

**Pull request merged**